ui/src/groups/autoJoinLureInvites.tsx

```diff
@@ -0,0 +1,13 @@
+import { Gangs } from '@/types/groups'
+import useGroupJoin from '@/groups/useGroupJoin';
+import cookies from 'browser-cookies';
+
+export default function AutoJoinLureInvites(pendingGangs: Gangs) {
```

Comment:
This should probably be converted into a hook that returns a "autoJoinLureInvites" function. 

`useGroupJoin` returns some functions that I think you probably want to use (maybe just the `join` function) rather than calling the hook directly.